backend/buildeR.API/appsettings.Development.json

```diff
@@ -10,5 +10,6 @@
   },
   "ElasticConfiguration": {
     "Uri": "http://localhost:9200"
-  }
+  },
+  "BaseUrl": "http://localhost:5050" 
```

Comment:
It won't work on PROD or any other PC

Reply:
So what should I do? Add appsettings.Production.json?

Reply:
Try to find how you can get server address